src/main/scala/org/hammerlab/guacamole/assembly/AssemblyUtils.scala

```diff
@@ -2,18 +2,40 @@ package org.hammerlab.guacamole.assembly
 
 import htsjdk.samtools.CigarOperator
 import org.apache.spark.Logging
+import org.bdgenomics.utils.cli.Args4jBase
 import org.hammerlab.guacamole.alignment.{AffineGapPenaltyAlignment, ReadAlignment}
 import org.hammerlab.guacamole.reads.MappedRead
 import org.hammerlab.guacamole.reference.{ContigSequence, ReferenceGenome}
 import org.hammerlab.guacamole.util.CigarUtils
 import org.hammerlab.guacamole.variants.ReferenceVariant
 import org.hammerlab.guacamole.windowing.SlidingWindow
+import org.kohsuke.args4j.{Option => Args4jOption}
 
 import scala.collection.JavaConversions._
 
 
 object AssemblyUtils extends Logging {
 
+  trait Arguments extends Args4jBase {
```

Comment:
Yea I'm fine doing that - I actually wasn't sure which way we were going, but followed `LociPartitionUtils` which has the `Arguments` trait nested. But I'll move this to `AsssemblyArgs`